### PR TITLE
MAUI-3858-Included UG for Migrate from Xamarin.Forms in MAUI DataForm 

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description :

I have included the following contents for Localization and RTL in MAUI DataForm:

* Namespaces 
* Initialize control
* Classes 
* Properties
* Enums
* Events
* Methods
* Upcoming Features

### CODE REVIEW

File: MAUI/DataForm/migration.md [hunk 1]
```diff
@@ -0,0 +1,961 @@
+---
+layout: post
+title: Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI SfDataForm
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:3]: Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI DataForm (SfDataForm)

File: MAUI/DataForm/migration.md [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+description: Learn here all about Migrating from Syncfusion Xamarin DataForm to Syncfusion .NET MAUI DataForm control and more.
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:4]: Learn here all about Migrating from Syncfusion Xamarin.Forms DataForm to Syncfusion .NET MAUI DataForm control.

File: MAUI/DataForm/migration.md [hunk 3]
```diff
@@ -0,0 +1,961 @@
...
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th></tr>
+<tr>
+<td>Syncfusion.SfDataForm.XForms</td>
+<td>Syncfusion.Maui.SfDataForm</td></tr>
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:22]: Syncfusion.Maui.DataForm

File: MAUI/DataForm/migration.md [hunk 4]
```diff
@@ -0,0 +1,961 @@
...
+this.Content = dataForm;
+
+{% endhighlight %}
+{% endtabs %}
+</td></tr>
+</table>
+
+## Classes 
+
+<table>
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:85]: Please all the new classes are addressed

New API - https://npmci.syncfusion.com/maui/api/internal-development/maui-dataform/api/toc.html#

Xamarin API - https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.html

File: MAUI/DataForm/migration.md [hunk 5]
```diff
@@ -0,0 +1,961 @@
...
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[]'| markdownify }}</td>
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:679]: add format link for xamarin

File: MAUI/DataForm/migration.md [hunk 6]
```diff
@@ -0,0 +1,961 @@
...
+
+<tr>
+<td>{{'[UpdateEditor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_UpdateEditor_System_String_)'| markdownify }}</td>
+<td>{{'[UpdateEditor]'| markdownify }}</td>
+<td>Updates the value of the specific property in corresponding editor.</td>
+</tr>
+</table>
+
+## Upcoming Features
+
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:959]: ## Upcoming Features

Floating label layout support
Keyboard interaction support for MacCatalyst, Android and iOS
View customization for group header.
Auto height support for DataFormViewItem
Programmatic scroll to a particular editor.
Themeing support for dataform.

Make better sentence for above

File: MAUI/DataForm/migration.md [hunk 7]
```diff
@@ -0,0 +1,961 @@
...
+## Known issues in .NET MAUI DataForm
+
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:961]: ## Known issues in .NET MAUI DataForm

- [MacCatalyst] - TimePicker popup is not working.
- [MacCatalyst, iOS] - Multiline editor placeholder text font size not changing based on editor font size.
- [MacCatalyst, iOS] – Date and Time picker text styles are not changing.
- [MacCatalyst, iOS] – Date picker minimum and maximum display dates are not working.
-  [MacCatalyst, iOS] - Text style not applied to radio button.
- [Android, iOS, MacCatalyst] - In RTL mode, the multiline text editor text is not showing from the right.
- [Windows] - Picker editor text style is not changing.
- [iOS] - Maximum input length is not working in Text and Password editors.
- [Windows] - Time picker format not changing.
- [MacCatalyst, iOS] - RTL FlowDirection is not working for Date and Time pickers.
- [MacCatalyst, iOS] - In RTL FlowDirection, the ScrollView scroll bar shows at the right.
- [Android] - in RTL mode Radio button's content does not arrange from the right.
- RTL flow direction not working for ComboBox and AutoComplete editors.
- IsReadOnly not working for AutoComplete editor.

File: MAUI/DataForm/migration.md [hunk 8]
```diff
@@ -0,0 +1,988 @@
...
+title: Migrate from Xamarin to .Net MAUI DataForm (SfDataForm) | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin.Forms DataForm to Syncfusion .NET MAUI DataForm control.
+platform: MAUI
+control: SfDataForm
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfDataForm to .NET MAUI SfDataForm 
+
+To make the migration from the Xamarin [Xamarin SfDataForm] to [.NET MAUI SfDataForm] easier, most of the APIs from the Xamarin SfDataForm were kept in the.NET MAUI SfDataForm. However, to maintain the consistency of API naming in the.NET MAUI SfDataForm, some of the APIs have been renamed. Please find the difference in the following topics.
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:12]: To make the migration from the  [Xamarin SfDataForm](https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform)

add link - https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform

Comment (karthikraja-arumugam) [reply MAUI/DataForm/migration.md:12]: [.NET MAUI SfDataForm] => `.NET MAUI SfDataForm`

Replace all applicable places

Comment (karthikraja-arumugam) [reply MAUI/DataForm/migration.md:12]: image

space issue

File: MAUI/DataForm/migration.md [hunk 9]
```diff
@@ -0,0 +1,987 @@
...
+</tr>
+</table>
+
+## Upcoming Features
+
+* Support for Floating label layout.
+* Keyboard interaction support for MacCatalyst, Android, and iOS.
+* Support for group header view customization.
+* Support for DataFormViewItem's auto height.
+* Support for programmatic scrolling to a specific editor,
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/migration.md:968]: remove comma

File: MAUI/DataForm/migration.md (hunk 1 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md]: Changed title

File: MAUI/DataForm/migration.md (hunk 3 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md]: changed to Syncfusion.Maui.DataForm

File: MAUI/DataForm/migration.md (hunk 4 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:84]: Checked all the new classes are addressed

File: MAUI/DataForm/migration.md (hunk 5 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md]: added format link for xamarin

File: MAUI/DataForm/migration.md (hunk 6 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:963]: Added upcoming features

File: MAUI/DataForm/migration.md (hunk 7 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:972]: Added known issues in .NET MAUI DataForm

File: MAUI/DataForm/migration.md (hunk 8 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md]: changes done

File: MAUI/DataForm/migration.md (hunk 9 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:968]: removed

File: MAUI/DataForm/migration.md [hunk 10]
```diff
@@ -0,0 +1,987 @@
...
+
+</td>
+<td>
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm x:Name="dataForm"/>
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/migration.md:64]: In example add DataObject

File: MAUI/DataForm/migration.md [hunk 11]
```diff
@@ -0,0 +1,987 @@
...
+
+## Properties
+
+### SfDataForm
+
+<table> 
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/migration.md:237]: Add example code for data object, with 2 more properites

File: MAUI/DataForm/migration.md [hunk 12]
```diff
@@ -0,0 +1,987 @@
...
+<tr>
+<td>Nil</td>
+<td>DefaultLayoutSettings</td>
+<td>Gets or sets the settings to customize default layout (Holds label and editor) settings.</td>
+</tr>
+</table>
+
+### DataFormTextStyle
+
+The properties of `Xamarin SfDataForm` [LabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html) are grouped to `DataFormTextStyle` in `.NET MAUI SfDataForm`
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/migration.md:320]: Add example

File: MAUI/DataForm/migration.md (hunk 10 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md]: DataObject added

File: MAUI/DataForm/migration.md (hunk 11 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:241]: Added example code

File: MAUI/DataForm/migration.md (hunk 12 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/migration.md:363]: Example added

File: MAUI/DataForm/migration.md [hunk 13]
```diff
@@ -0,0 +1,1085 @@
...
+title: Migrate from Xamarin to .Net MAUI DataForm (SfDataForm) | Syncfusion
+description: Learn about Migrating from Syncfusion Xamarin.Forms DataForm to Syncfusion .NET MAUI DataForm control.
+platform: MAUI
+control: SfDataForm
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfDataForm to .NET MAUI SfDataForm 
+
+To make the migration from the Xamarin [Xamarin SfDataForm](https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform) to `.NET MAUI SfDataForm` easier, most of the APIs from the Xamarin SfDataForm were kept in the `.NET MAUI SfDataForm` However, to maintain the consistency of API naming in the `.NET MAUI SfDataForm`, some of the APIs have been renamed. Please find the difference in the following topics.
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:12]: To make the migration from the Xamarin [Xamarin SfDataForm](https://www.syncfusion.com/xamarin-ui-controls/xamarin-dataform) to `.NET MAUI SfDataForm` easier, most of the APIs from the Xamarin SfDataForm were kept in the `.NET MAUI SfDataForm.` However, to maintain the consistency of API naming in the `.NET MAUI SfDataForm,` some of the APIs have been renamed. Please find the difference in the following topics.

File: MAUI/DataForm/migration.md [hunk 14]
```diff
@@ -0,0 +1,1085 @@
...
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[KeyBoard](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTextItemBase.html#Syncfusion_XForms_DataForm_DataFormTextItemBase_KeyBoard)'| markdownify }}</td>
+<td>Keyboard</td>
+<td>Gets or sets the type of keyboard used to display in the text, multiline and password editor.</td>
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:740]: <td>Gets or sets the type of keyboard used to display the text, multiline, and password editor.</td>

File: MAUI/DataForm/migration.md [hunk 15]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTimeItem.html#Syncfusion_XForms_DataForm_DataFormTimeItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for Time Picker.</td>
+</tr>
+</table>
+
+### DataFormListItem
+
+The common properties of `Xamarin SfDataForm` list items (` DataFormPickerItem`, ` DataFormAutoCompleteItem`, ` DataFormDropDownItem` and ` DataFormRadioGroupItem` ) are added to [DataFormListItem] class in `.NET MAUI SfDataForm`
```
Comment (florenceliliana) [inline MAUI/DataForm/migration.md:789]: The common properties of `Xamarin SfDataForm` list items (` DataFormPickerItem,` ` DataFormAutoCompleteItem,` ` DataFormDropDownItem` and ` DataFormRadioGroupItem` ) are added to the [DataFormListItem] class in ` the .NET MAUI SfDataForm.`

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/DataForm/migration.md [hunk 16]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[ValidatingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidatingEventArgs.html)'| markdownify }}</td>
+<td>DataFormValidateFormEventArgs</td>
+<td>Provides data for ValidateForm event.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>DataFormValidationMode</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:228]: <td>Defines the constants for the ValidationMode that specifies when the user input should be validated for the data object.</td>

File: MAUI/DataForm/migration.md [hunk 17]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[ColumnCount](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_ColumnCount)'| markdownify }}</td>
+<td>ColumnCount</td>
+<td>Gets or sets the value representing the number of editors to be arranged per row in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[SourceProvider](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_SourceProvider)'| markdownify }}</td>
+<td>ItemsSourceProvider</td>
+<td>Gets or sets the value to get the source for picker type editors.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:321]: <td>Gets or sets the value to get the source for the picker type editors.</td>

File: MAUI/DataForm/migration.md [hunk 18]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>{{'[AutoGenerateItems](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_AutoGenerateItems)'| markdownify }}</td>
+<td>AutoGenerateItems</td>
+<td>Gets or sets a value indicating whether auto generate the DataFormItems in the data form.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:327]: <td>Gets or sets a value indicating whether to auto-generate the DataFormItems in the data form.</td>

File: MAUI/DataForm/migration.md [hunk 19]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>EditorTextStyle</td>
+<td>Gets or sets the style of editor text, that used to customize the text color, font, font size, font family and font attributes.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:333]: <td>Gets or sets the style of the editor text used to customize the text color, font, font size, font family, and font attributes.</td>

File: MAUI/DataForm/migration.md [hunk 20]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>LabelTextStyle</td>
+<td>Gets or sets the style of label text, that used to customize the text color, font, font size, font family and font attributes.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:339]: <td>Gets or sets the label text style used to customize the text color, font, font size, font family, and font attributes.</td>

File: MAUI/DataForm/migration.md [hunk 21]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ErrorLabelTextStyle</td>
+<td>Gets or sets the style of error label text, that used to customize the text color, font, font size, font family and font attributes.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:345]: <td>Gets or sets the style of error label text used to customize the text color, font, font size, font family, and font attributes.</td>

File: MAUI/DataForm/migration.md [hunk 22]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ValidMessageLabelTextStyle</td>
+<td>Gets or sets the style of validation label text, that used to customize the text color, font, font size, font family and font attributes.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:351]: <td>Gets or sets the style of the validation label text used to customize the text color, font size, font family, and font attributes.</td>

File: MAUI/DataForm/migration.md [hunk 23]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DefaultLayoutSettings</td>
+<td>Gets or sets the settings to customize default layout (Holds label and editor) settings.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:357]: <td>Gets or sets the settings to customize the default layout (Holds label and editor) settings.</td>

File: MAUI/DataForm/migration.md [hunk 24]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+</table>
+
+### DataFormTextStyle
+
+The properties of `Xamarin SfDataForm` [LabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html) are grouped to `DataFormTextStyle` in `.NET MAUI SfDataForm`
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:363]: The properties of the `Xamarin SfDataForm` [LabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html) are grouped to the `DataFormTextStyle` in `.NET MAUI SfDataForm.`

File: MAUI/DataForm/migration.md [hunk 25]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>Nil</td>
+<td>TextColor</td>
+<td>Gets or sets the editor text color in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[FontSize](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html#Syncfusion_XForms_DataForm_LabelStyle_FontSize)'| markdownify }}</td>
+<td>FontSize</td>
+<td>Gets or sets the double value that represents text font size in the data form.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:435]: <td>Gets or sets the double value that represents the text font size in the data form.</td>

File: MAUI/DataForm/migration.md [hunk 26]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>{{'[FontFamily](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelStyle.html#Syncfusion_XForms_DataForm_LabelStyle_FontFamily)'| markdownify }}</td>
+<td>FontFamily</td>
+<td>Gets or sets the string, that represents text font family in the data form.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:441]: <td>Gets or sets the string that represents the text font family in the data form.</td>

File: MAUI/DataForm/migration.md [hunk 27]
```diff
@@ -0,0 +1,1085 @@
...
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[RowSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItemBase.html#Syncfusion_XForms_DataForm_DataFormItemBase_RowSpan)'| markdownify }}</td>
+<td>RowSpan</td>
+<td>Gets or sets the number of rows that this item spans which is used to increase item's height.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:464]: <td>Gets or sets the number of rows this item spans, which increases the item's height.</td>

File: MAUI/DataForm/migration.md [hunk 28]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[ShowLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ShowLabel)'| markdownify }}</td>
+<td>ShowLabel</td>
+<td>Gets or sets a value indicating whether to show the label to the editor.</td>
+</tr>
+
+<tr>
+<td>{{'[IsReadOnly](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_IsReadOnly)'| markdownify }}</td>
+<td>IsReadOnly</td>
+<td>Gets or sets a value indicating whether editor can be enabled or disabled.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:540]: <td>Gets or sets a value indicating whether the editor can be enabled or disabled.</td>

File: MAUI/DataForm/migration.md [hunk 29]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>Nil</td>
+<td>Background</td>
+<td>Gets or sets the background to the data form item layout which is used to render label and editor for the data field.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DefaultLayoutSettings</td>
+<td>Gets or sets the value of layout (holds label and editor) settings which is used to customize the label position, label width and editor width.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:576]: <td>Gets or sets the value of the layout (holds label and editor) settings used to customize the label position, label width, and editor width.</td>

File: MAUI/DataForm/migration.md [hunk 30]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[PropertyInfo](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_PropertyInfo)'| markdownify }}</td>
+<td>PropertyInfo</td>
+<td>Gets the property info of the data field.</td>
+</tr>
+</table>
+
+### DataFormDefaultLayoutSettings
+
+Following properties of `Xamarin SfDataForm` is moved to `DataFormDefaultLayoutSettings` class of `.NET MAUI SfDataForm`.
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:607]: The following properties of the `Xamarin SfDataForm` is moved to the `DataFormDefaultLayoutSettings` class of `.NET MAUI SfDataForm.`

File: MAUI/DataForm/migration.md [hunk 31]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>{{'[LabelPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_LabelPosition)'| markdownify }}</td>
+<td>LabelPosition</td>
+<td>Gets or sets the label position for DataFormItem to define the position of the label which associated with the editor.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:619]: <td>Gets or sets the label position for DataFormItem to define the position of the label associated with the editor.</td>

File: MAUI/DataForm/migration.md [hunk 32]
```diff
@@ -0,0 +1,1085 @@
...
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Format](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormTimeItem.html#Syncfusion_XForms_DataForm_DataFormTimeItem_Format)'| markdownify }}</td>
+<td>Format</td>
+<td>Gets or sets the time format for Time Picker.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:783]: <td>Gets or sets the time format for the Time Picker.</td>

File: MAUI/DataForm/migration.md (hunk 15 above)
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:789]: The common properties of the `Xamarin SfDataForm` list items (` DataFormPickerItem,` ` DataFormAutoCompleteItem,` ` DataFormDropDownItem,` and ` DataFormRadioGroupItem` ) are added to the [DataFormListItem] class in `.NET MAUI SfDataForm.`

File: MAUI/DataForm/migration.md [hunk 33]
```diff
@@ -0,0 +1,1085 @@
...
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ItemsSource](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_ItemsSource)'| markdownify }}</td>
+<td>ItemsSource</td>
+<td>Gets or sets the collection which will get displayed in picker where the user can select from available items.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:800]: <td>Gets or sets the collection, which will be displayed in the picker where the user can select from available items.</td>

File: MAUI/DataForm/migration.md [hunk 34]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>{{'[DisplayMemberPath](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormPickerItem.html#Syncfusion_XForms_DataForm_DataFormPickerItem_DisplayMemberPath)'| markdownify }}</td>
+<td>DisplayMemberPath</td>
+<td>Gets or sets a path to a value on the source object that will get displayed in picker.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:806]: <td>Gets or sets a path to a value on the source object that will get displayed in the picker.</td>

File: MAUI/DataForm/migration.md [hunk 35]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>Nil</td>
+<td>IsEditable</td>
+<td>Gets or sets a value indicating whether the ComboBox can be edited.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for ComboBox editor.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:833]: <td>Gets or sets the text search mode for the ComboBox editor.</td>

File: MAUI/DataForm/migration.md [hunk 36]
```diff
@@ -0,0 +1,1085 @@
...
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[AutoCompleteMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormAutoCompleteItem.html#Syncfusion_XForms_DataForm_DataFormAutoCompleteItem_AutoCompleteMode)'| markdownify }}</td>
+<td>TextSearchMode</td>
+<td>Gets or sets the text search mode for auto complete editor.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:849]: <td>Gets or sets the text search mode for the auto complete editor.</td>

File: MAUI/DataForm/migration.md [hunk 37]
```diff
@@ -0,0 +1,1085 @@
...
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ColumnSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ColumnSpan)'| markdownify }}</td>
+<td>ColumnSpan</td>
+<td>Gets or sets the number of column that this item spans to increase item's width.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:865]: <td>Gets or sets the number of columns this item spans to increase the item's width.</td>

File: MAUI/DataForm/migration.md [hunk 38]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>{{'[RowSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_RowSpan)'| markdownify }}</td>
+<td>RowSpan</td>
+<td>Gets or sets the number of rows that this item spans which is used to increase item's height.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:871]: <td>Gets or sets the number of rows this item spans, which is used to increase the item's height.</td>

File: MAUI/DataForm/migration.md [hunk 39]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[ShowLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ShowLabel)'| markdownify }}</td>
+<td>ShowLabel</td>
+<td>Gets or sets a value indicating whether to show the label to the DataForm editor.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidMessage](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DisplayOptionsAttribute.html#Syncfusion_XForms_DataForm_DisplayOptionsAttribute_ValidMessage)'| markdownify }}</td>
+<td>ValidMessage</td>
+<td>Gets or sets a string which will get displayed when user enters valid value for the data field as success message.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:895]: <td>Gets or sets a string displayed when the user enters a valid value for the data field as a success message.</td>

File: MAUI/DataForm/migration.md [hunk 40]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[MaxYear](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DateRangeAttribute.html#Syncfusion_XForms_DataForm_DateRangeAttribute_MaxYear)'| markdownify }}</td>
+<td>Nil</td>
+<td>Gets or sets the maximum year of a date.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DisplayFormat</td>
+<td>Gets or sets the attribute date display format of date editor.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:947]: <td>Gets or sets the attribute date display format of the date editor.</td>

File: MAUI/DataForm/migration.md [hunk 41]
```diff
@@ -0,0 +1,1085 @@
...
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[CommitMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.CommitMode.html)'| markdownify }}</td>
+<td>DataFormCommitMode</td>
+<td>Defines the constants for CommitMode that specifies when the value entered by user should be saved to the data object.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:978]: <td>Defines the constants for the CommitMode that specify when the value entered by the user should be saved to the data object.</td>

File: MAUI/DataForm/migration.md [hunk 42]
```diff
@@ -0,0 +1,1085 @@
...
+</tr>
+
+<tr>
+<td>{{'[LabelPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.LabelPosition.html)'| markdownify }}</td>
+<td>DataFormLabelPosition</td>
+<td>Defines the constants for label position that specifies whether to layout the label at left or top of editor.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:984]: <td>Defines the constants for the label position that specifies whether to lay out the label at the left or top of the editor.</td>

File: MAUI/DataForm/migration.md [hunk 43]
```diff
@@ -0,0 +1,1085 @@
...
+<tr>
+<td>{{'[SuggestionMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SuggestionMode.html)'| markdownify }}</td>
+<td>DataFormTextSearchMode</td>
+<td>Specifies the type of the text search mode in ComboBox and AutoComplete.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationMode](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.ValidationMode.html)'| markdownify }}</td>
+<td>DataFormValidationMode</td>
+<td>Defines the constants for ValidationMode that specifies when the user input should be validated for data object.</td>
```
Comment (Chrispineagunja) [inline MAUI/DataForm/migration.md:996]: <td>Defines the constants for the ValidationMode that specifies when the user input should be validated for the data object.</td>

**Pull request merged**